src/helpers.py

```diff
@@ -0,0 +1,53 @@
+import src.checks as checks
+
+def IMPORTANT_CHECKS():
+    return [checks.valid_hostname,checks.prohibited_networks,checks.nameserver_reachability,
+                   checks.dns_test_recursion, checks.answer_authoritatively,checks.same_source_address]
+                   
+def ADDITIONAL_CHECKS():
+    return [checks.consistency_glue_authoritative, checks.consistent_delegation_zone,
+    checks.consistent_authoritative_nameservers,
+    checks.minimal_ns,checks.network_diversity,checks.truncref]
+
+def get_details(result, id):
+    return {"id" : id, "result" : result.get("result"), "description" : result.get("description"), "details" : result.get("details")}
```

Comment:
The format for the JSON response is not correct, there's should be a "key" field

Reply:
I talked w/ Kasper and we agreed that the name "details" is more clear since we retrieve stuff from the details label on the dictionary, similar work is ongoing on front-end(using the keyword details than key)